### PR TITLE
chore (refs DPLAN-12198): Add missing Hooks/Selectors.

### PR DESCRIPTION
**Pull request by ahmad-demos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-12198
https://demoseurope.youtrack.cloud/issue/DPLAN-12198/Konnen-wir-data-cybuttonRowsaveButton-und-data-cybuttonRowabortButton-zu-data-cysaveButton-und-data-cyabortButton-umbenennen#focus=Comments-4-51786.0-0

- Add missing Hooks/Selectors.

### CODE REVIEW

File: client/js/components/procedure/admin/AdministrationPlaces.vue [hunk 1]
```diff
@@ -56,6 +56,7 @@
           }" />
         <dp-button-row
           :busy="isLoading"
+          data-cy="addNewPlaceForm"
```
Comment (spiess-demos) [inline client/js/components/procedure/admin/AdministrationPlaces.vue:59]: `addNewPlace` would suffice i guess.

File: client/js/components/statement/splitStatement/SideBar.vue [hunk 2]
```diff
@@ -155,6 +155,7 @@
 
     <dp-button-row
       class="p-2"
+      data-cy="assignedTags"
```
Comment (spiess-demos) [inline client/js/components/statement/splitStatement/SideBar.vue:158]: i think that the kind of action should be part of the hook name, like `saveAssignedTags`, what do you think?

**Review by spiess-demos — CHANGES_REQUESTED** [review]
some nitpicky naming questions

File: client/js/components/procedure/admin/AdministrationPlaces.vue (hunk 1 above)
Comment (ahmad-demos) [reply client/js/components/procedure/admin/AdministrationPlaces.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/3564/commits/50933cff9939c7d48f4618ceecaf819e46043a8e
Thanks!

File: client/js/components/statement/splitStatement/SideBar.vue (hunk 2 above)
Comment (ahmad-demos) [reply client/js/components/statement/splitStatement/SideBar.vue:158]: DpButtonRow component in demosplan-ui has 2 buttons anyway, with:
```
:data-cy="`${dataCy}:saveButton`" 
:data-cy="`${dataCy}:abortButton`"
```
then here with my changes:
`data-cy="assignedTags:saveButton"` and `data-cy="assignedTags:abortButton"`

Comment (ahmad-demos) [reply client/js/components/statement/splitStatement/SideBar.vue:158]: @spiess-demos

Comment (spiess-demos) [reply client/js/components/statement/splitStatement/SideBar.vue:158]: ok, i did not realize that. thanks for the explanation!

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**